nvi-commons/src/testFixtures/java/no/sikt/nva/nvi/common/queue/FakeSqsClient.java

```diff
@@ -83,12 +76,21 @@ public void deleteMessage(String queueUrl, String receiptHandle) {
     deleteMessages.add(request);
   }
 
-  private static List<String> extractSuccessfulEntryIds(SendMessageBatchResponse response) {
-    return response.successful().stream().map(SendMessageBatchResultEntry::id).toList();
+  private ReceiveMessageRequest createReceiveRequest(String queueUrl, int maxNumberOfMessages) {
+    return ReceiveMessageRequest.builder()
+        .queueUrl(queueUrl)
+        .maxNumberOfMessages(maxNumberOfMessages)
+        .build();
   }
 
-  private static List<String> extractFailedEntryIds(SendMessageBatchResponse response) {
-    return response.failed().stream().map(BatchResultErrorEntry::id).toList();
+  private NviReceiveMessageResponse createResponse(ReceiveMessageResponse receiveMessageResponse) {
+    return new NviReceiveMessageResponse(
+        receiveMessageResponse.messages().stream()
+            .map(
+                m ->
+                    new NviReceiveMessage(
+                        m.body(), m.messageId(), m.attributesAsStrings(), m.receiptHandle()))
```

Comment:
Pull this to its own method, or create relevant constructor

Reply:
Thanks! I'll keep it in mind and do it in a separate PR. Tried to just copy/paste as much as possible in this one.